src/schemas.ts

```diff
@@ -17,6 +17,12 @@ const blockchainAddress: JSONSchemaType<Address> = {
   maxLength: 42,
 };
 
+const resourceType: JSONSchemaType<string> = {
```

Comment:
Consider constraining the allowed values to the current acceptable integers 

I believe this is possible with JSONSchemaType (if not ignore)

Reply:
I defined pattern for resourceType: https://github.com/zer0-os/data-store-api/pull/27/commits/3d44b22b660d6cfac8c9607b733d76910a9c1bbf#diff-8f5fdeb1cfe9f99fbc1c0a2c351f90ef3531ecdeefa8d0f9a4f0f4127c67f2aaR22